src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -167,9 +178,18 @@ struct ApparentHorizon {
     // Put Strahlkorper and its ComputeItems, FastFlow,
     // and verbosity into a new DataBox.
     // Put the initial guess also into PreviousStrahlkorper.
-    Initialization::mutate_assign<simple_tags>(
-        box, options.initial_guess, options.fast_flow, options.verbosity,
-        options.initial_guess);
+    if constexpr (std::is_same_v<Frame, ::Frame::Inertial>) {
+      Initialization::mutate_assign<simple_tags>(
+          box, options.initial_guess, options.fast_flow, options.verbosity,
+          options.initial_guess);
+    } else {
+      // ... and also default-initialize Strahlkorper in inertial
+      // frame, if we are finding the Strahlkorper in a non-inertial
+      // frame.
+      Initialization::mutate_assign<simple_tags>(
+          box, options.initial_guess, options.fast_flow, options.verbosity,
+          options.initial_guess, Strahlkorper<::Frame::Inertial>{});
+    }
   }
 
   template <typename Metavariables, typename DbTags, typename TemporalId>
```

Comment:
shouldn't there be something added to a test somewhere to test the changes made in this PR?